frontend/src/i18n/index.ts

```diff
@@ -0,0 +1,7 @@
+import English from './locales/en-us.json';
```

Comment:
Presumably this file needs to be updated to add a new language? The README should have some docs on how to add new languages too

Reply:
Yup, added instructions here: https://github.com/ScopeLift/umbra-protocol/pull/331#discussion_r875145605

Reply:
Yes, added the instructions here: https://github.com/ScopeLift/umbra-protocol/pull/331#discussion_r875079735